### ISSUE TITLE
Orphaned Labels on Add to Collection Modal

### ISSUE THREAD

**Issue by mtribone** [projecthydra/sufia#970]
Technically not orphaned per the accessibility tool, but we have a label mismatch with their respective inputs. The ID for inputs is prefixed with "id_" and the for attribute of the label does not have this prefix. These should match for better accessibility compliance.

![screen shot 2015-03-17 at 3 53 17 pm](https://cloud.githubusercontent.com/assets/4163828/6696237/45a6924c-ccbe-11e4-8797-ae77a2f66526.png)

![screen shot 2015-03-17 at 3 57 51 pm](https://cloud.githubusercontent.com/assets/4163828/6696261/68da62de-ccbe-11e4-90e1-1492833ea0bb.png)
